### DISCUSSION TITLE
Alternative function key layout

### DISCUSSION THREAD

**Discussion by ghost** [manna-harbour/miryoku#91] [Q&A] [answered]
I have found the current function key layout to be a bit complicated for me. While mirroring the num layer is quite nice, I think it leaves the pinky keys in an weird spot of having to memorize the order. Coming from a 75%-or-larger keyboard, I have found the function key groupings on those keyboards to be more intuitive. I've adopted this to the miryoku layout and have been using it for quite some time now. I think it works nicely while still adhering to the general principles. I would like to propose the alternative function key layout so that others may benefit from it.

**Current layout**

```
F12 F7 F8 F9
F11 F4 F5 F6
F10 F1 F2 F3
```

**Proposed layout**

```
F9 F10 F11 F12
F5 F6  F7  F8
F1 F2  F3  F4
```

**Comment by manna-harbour** [comment, marked as answer]
Thanks for the suggestion!

It's now possible to substitute layers by defining the replacement in `custom_config.h`. See https://github.com/manna-harbour/miryoku/discussions/85. If you'd like to share your alternative so that others can use it directly, you could post the `#define` from `custom_config.h` to [Show and Tell](https://github.com/manna-harbour/miryoku/discussions/categories/show-and-tell).

I may be adding alternative Num, Sym, and Fun layouts at some point, but first `miryoku_layer_selection.h` needs to be auto-generated or replaced with a different mechanism.
